### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by mribeirodantas**
- Update file extension for YAML parsing example
- Update channel factory for example 6.3.4
- Improve wording for single script block in the process
- Remove repeated subsection for file/path
- Improves Value Channel subsection

### CODE REVIEW

File: asciidocs/channels.adoc [hunk 1]
```diff
@@ -51,26 +51,117 @@ ch.view()
...
+process SUM {
+    input:
+    val x
+    val y
+    output:
+    stdout
+    script:
+    """
+    echo \$(($x+$y))
+    """
```
Comment (abhi18av) [inline asciidocs/channels.adoc:72]:
```suggestion
    val y
    
    output:
    stdout
    
    script:
    """
    echo \$(($x+$y))
    """
```

Comment (abhi18av) [reply asciidocs/channels.adoc:72]: Made the example a bit less dense.

File: asciidocs/channels.adoc [hunk 2]
```diff
@@ -51,26 +51,117 @@ ch.view()
...
+4
+
+3
+
+2
+----
+
+Besides, in many situations Nextflow will implicitly convert variablers to value channels. When you run a
+process with a parameter (`params.example`), the content is just a string, for example, but when the process
+will receive it as a value channel.
```
Comment (abhi18av) [inline asciidocs/channels.adoc:164]:
```suggestion
Besides, in many situations, Nextflow will implicitly convert variables to value channels when they are used in a process invocation. For example, when you invoke a process with a pipeline parameter (`params.example`) which has a string value, it is automatically cast into a value channel.
```

File: asciidocs/processes.adoc [hunk 3]
```diff
@@ -65,7 +65,7 @@ process < name > {
 
 The `script` block is a string statement that defines the command to be executed by the process.
 
-A process contains one and only one `script` block. It must be the last statement when the process contains input and output declarations.
+A process can execute one and only one `script` block. It must be the last statement when the process contains input and output declarations.
```
Comment (abhi18av) [inline asciidocs/processes.adoc:68]:
```suggestion
A process can execute only one `script` block. It must be the last statement when the process contains input and output declarations.
```

File: asciidocs/processes.adoc [hunk 4]
```diff
@@ -355,39 +355,12 @@ workflow {
 }
 ----
 
-WARNING: When a process declares an input file, the corresponding channel elements
+WARNING: In the past, the file qualifier was used for files, but the path qualifier
```
Comment (abhi18av) [inline asciidocs/processes.adoc:358]:
```suggestion
WARNING: In the past, the `file` qualifier was used for files, but the `path` qualifier
```

**Comment by abhi18av** [comment]
@mribeirodantas , I wasn't sure if I should just push these changes directly therefore you can make use of the `Commit changes` functionality to include the ones which make sense.

image

**Comment by mribeirodantas** [comment]
> @mribeirodantas , I wasn't sure if I should just push these changes directly therefore you can make use of the `Commit changes` functionality to include the ones which make sense.
> 
Thanks for the great review, @abhi18av ! 🥇

**Comment by mribeirodantas** [comment]
@chriswyatt1 LGTM?

**Comment by chriswyatt1** [comment]
> @chriswyatt1 LGTM?

I am reviewing now. Will let ya know when complete

**Review by chriswyatt1 — APPROVED** [review]
Awesome, great to have the dsl1 poison pill example in there!. LGTM

**Pull request merged**